content/docs/studio/user-guide/projects-and-experiments/run-experiments.md

```diff
@@ -146,27 +146,59 @@ automatically be created from the new branch to the base branch. Now, click on
 
 ![](https://static.iterative.ai/img/studio/cml_commit.png)
 
-At this point, the new experiment appears in the project's experiment table. If
-you just committed to a new branch, then a new pull request will also have been
-created from the new branch to the base branch.
+## What happens after you submit a new experiment
+
+Iterative Studio will
+[create the Git commit and pull request](#git-commit-and-pull-request-are-created).
+This will [invoke your model training process](#model-training-is-invoked) (if
+it is set up). You can
+[track the results in real-time](#live-metrics-and-plots-are-tracked). And you
+can [save the results in Git](#metrics-plots-and-reports-are-saved-in-git).
+
+### Git commit (and pull request) are created
+
+Iterative Studio will create a Git commit with the changes you submitted. This
+commit appears in the project's experiment table. If you had specified a new
```

Comment:
(Same issue:) A regular commit appears as an experiment?

Reply:
p.s. I guess it's related to calling all these things "rows" both in the UI and docs. If that update is planned, we can wait for the core update to clarify all these texts.

Reply:
yes